dags/aggregates_python_helpers.py

```diff
@@ -40,28 +32,33 @@ def validation(file_path):
     # Read data from CSV into a pandas DataFrame
     df = pd.read_csv(file_path, delimiter='#', encoding='ISO-8859-2')
     
+    df['terc'] = df['terc'].apply(lambda x: str(int(x)) if pd.notnull(x) and str(x).replace('.0', '').isdigit() else str(x))
+    
     # Convert pandas DataFrame to Great Expectations PandasDataset for validation
     dataset = PandasDataset(df)
 
     # Define expectations
     rom_set = create_roman_set()
     dataset.expect_column_values_to_match_regex('data_wplywu_wniosku_do_urzedu', r'^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}$')
     dataset.expect_column_values_to_be_in_set('kategoria', rom_set)
+    dataset.expect_column_values_to_match_regex('terc', r'^\d{7}$')
```

Comment:
could we check if for example the rodzaj_zamierzenia is from the 4 possible options ?